lib/charms/ovn_charm.py

```diff
@@ -233,6 +242,92 @@ def mlockall_disabled(self):
                 self._disable_mlockall = True
         return self._disable_mlockall
 
+    @property
+    def bridge_port_interfaces(self):
```

Comment:
The naming of this property might be a bit confusing, the configuration option it shadows is named `bridge-interface-mappings`.

While I believe we cannot create a new property with the same name as the actual config option, would it make sense to base the name on the config option with either a prefix or suffix?